content/content-strategy/manage-content-requests/create-content/workflow-process.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: text/textblock
+hidden: false
```

Comment:
This param shouldn't be here. Recommend: replace with `section` param and specify what heading this relates to.

Reply:
Trisha: done